### PR TITLE
PLATFORM-2770 Make forumActivityModule internal

### PR DESCRIPTION
**Pull request by themech**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ForumController.class.php#L331](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L331): Remove this unused "$key" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [ForumController.class.php#L345](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L345): Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)
1. MINOR [ForumController.class.php#L7](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L7): Remove this method "__construct" to simply inherit it. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)
1. MINOR [ForumController.class.php#L52](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L52): Define a constant instead of duplicating this literal "activeThreads" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumController.class.php#L53](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L53): Define a constant instead of duplicating this literal "isTopicPage" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumController.class.php#L89](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L89): Immediately return this expression instead of assigning it to the temporary variable "$topicTitle". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. MINOR [ForumController.class.php#L195](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L195): Define a constant instead of duplicating this literal "title" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumController.class.php#L295](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L295): Define a constant instead of duplicating this literal "forum_sort_" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. INFO [ForumController.class.php#L76](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L76): Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
1. INFO [ForumController.class.php#L171](https://github.com/Wikia/app/blob/073c4b11be7faacbc7888af8b8eee671e3d70bee/extensions/wikia/Forum/ForumController.class.php#L171): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

File: extensions/wikia/Forum/ForumController.class.php [hunk 1]
```diff
@@ -310,6 +310,10 @@ protected function getSortingSelected() {
 	}
 
 	public function forumActivityModule() {
+		if ( !$this->request->isInternal()) {
```
Comment (Grunny) [inline extensions/wikia/Forum/ForumController.class.php]: Super minor: missing space before closing parenthesis per conventions.

**Review by Grunny — APPROVED** [review]
LGTM.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [ForumController.class.php#L331](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L331): Remove this unused "$key" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [ForumController.class.php#L345](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L345): Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)
1. MINOR [ForumController.class.php#L7](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L7): Remove this method "__construct" to simply inherit it. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)
1. MINOR [ForumController.class.php#L52](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L52): Define a constant instead of duplicating this literal "activeThreads" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumController.class.php#L53](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L53): Define a constant instead of duplicating this literal "isTopicPage" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumController.class.php#L89](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L89): Immediately return this expression instead of assigning it to the temporary variable "$topicTitle". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. MINOR [ForumController.class.php#L195](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L195): Define a constant instead of duplicating this literal "title" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. MINOR [ForumController.class.php#L295](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L295): Define a constant instead of duplicating this literal "forum_sort_" 3 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
1. INFO [ForumController.class.php#L76](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L76): Complete the task associated to this "TODO" comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)
1. INFO [ForumController.class.php#L171](https://github.com/Wikia/app/blob/5eee4818009fbe36ae65a6ce110026125b6577ae/extensions/wikia/Forum/ForumController.class.php#L171): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Pull request merged**